src/platform-includes/source-context/java.mdx

```diff
@@ -277,6 +332,18 @@ Next, upload that source bundle to Sentry:
 sentry-cli debug-files upload --type jvm path/to/bundle
 ```
 
+<PlatformSection supported={["android"]}>
+
+### Uploading including the Native Source Bundle
+
+Include the native source bundle with `--include-sources` and upload it to Sentry:
+
+```
+sentry-cli debug-files upload --include-sources /path/to/files --type jvm path/to/bundle
```

Comment:
hope i got the command right here

Reply:
I don't think it works this way, at least the Android Gradle Plugin performs two uploads: One for JVM and one for Native sources / symbols - I've added a comment below how it works.